runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -48,6 +52,16 @@
 public class TearsOfGuthixPlugin extends Plugin
 {
 	private static final int TOG_REGION = 12948;
+	private static final ImmutableList<Skill> TOG_SKILLS = ImmutableList.of(

```

Comment:
Since it's important this contains all non-`OVERALL` skills, it might be worth writing a small test to ensure that's the case.